src/h5web/App.test.tsx

```diff
@@ -35,22 +35,15 @@ test('renders with root group selected', async () => {
 });
 
 test('switch between "display" and "inspect" modes', async () => {
-  const {
-    findByRole,
-    findByText,
-    getByRole,
-    queryByRole,
-    queryByText,
-  } = renderApp();
+  const { findByRole, getByRole, queryByRole } = renderApp();
 
   const inspectBtn = await findByRole('tab', { name: 'Inspect' });
   const displayBtn = getByRole('tab', { name: 'Display' });
 
   // Switch to "inspect" mode
   fireEvent.click(inspectBtn);
-
-  const noVisText1 = queryByText('Nothing to visualize');
-  expect(noVisText1).not.toBeInTheDocument();
+  const visSelector1 = queryByRole('tablist', { name: 'Visualization' });
+  expect(visSelector1).not.toBeInTheDocument();
```

Comment:
Funnily, I had to change this as the root node has now a visualization :slightly_smiling_face: